src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -91,23 +113,27 @@ export default class SimidMapCreative extends BaseSimidCreative {
    *   category and can be specified by the advertisers. If the value is not specified, 
    *   then BUTTON_LABEL's value will default to Location.
    * @private 
-   */   
+   */
   specifyButtonFeatures_(buttonLabel = DEFAULT_BUTTON_LABEL) {
     const findNearestButton = document.getElementById('findNearest');
     findNearestButton.innerText = FIND_NEAREST_TEMPLATE_TEXT + buttonLabel;
     findNearest.onclick = () => this.prepareCreative_();
   }
 
+  /**
+   * 
+   * @private 
+   */
```

Comment:
fix this jsdoc.

If all youre gonna include is a single tag, you can use the condensed form 
`/** @private */`
but probably good to include a method description as well